infra/base-images/base-runner/coverage

```diff
@@ -50,8 +53,9 @@ NPROC=$(nproc)
 
 function run_fuzz_target {
   local target=$1
-  local profraw_file="$target.profraw"
-  local profdata_file="$target.profdata"
+  local profraw_file="$DUMPS_DIR/$target.%1m.profraw"
```

Comment:
Can you add a comment to describe the %1m use.

Reply:
Done